databricks-iris/{{ cookiecutter.repo_name }}/requirements.txt

```diff
@@ -2,7 +2,7 @@ ipython>=8.10
 jupyterlab>=3.0
 notebook
 kedro~={{ cookiecutter.kedro_version }}
-kedro-datasets[spark.SparkDataset, pandas.ParquetDataset]>=1.0
+kedro-datasets[spark, pandas, spark.SparkDataset, pandas.ParquetDataset]>=1.0
```

Comment:
Is there an argument we should encourage the use of [databricks.ManagedTableDataSet](https://kedro.org/blog/managed-delta-tables-kedro-dataset) (even if it's commented out in the catalog) since it highlights Kedro's compatibility with Unity Catalog + Delta lake?

Reply:
Agree 👆

But perhaps better saved for another PR? :)

Reply:
absolutely, this is a really important and frankly overdue piece of work and I'm just thinking about how to take it further, later!

Reply:
For reference, +1 on this!